runtimevar/driver/driver.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package driver provides the interface for providers of runtimevar.  This serves as a contract
-// of how the runtimevar API uses a provider implementation.
+// Package driver defines an interface that the runtimevar package uses to
+// interact with the underlying runtime configuration services.
```

Comment:
Similarly to https://golang.org/pkg/database/sql/driver/, suggest the following rewrite (here and in all other APIs' driver.go): 
"Package driver defines interfaces to be implemented by runtime configuration drivers and to be used by package runtimevar to interact with the underlying services. Most code should use package runtimevar."

Reply:
I updated them all to be consistent, with a slight rewording of your suggestion. PTAL.